integritybackend/actions.py

```diff
@@ -149,9 +149,12 @@ def archive(self, zip_path: str, org_config: dict, collection_id: str):
                     authsign_server_url,
                     authsign_auth_token,
                 )
-                _logger.info(
-                    f"Content signed by authsign server: {content_authsign_path}"
-                )
+                if content_authsign_path != None:
+                    _logger.info(
+                        f"Content signed by authsign server: {content_authsign_path}"
+                    )
+                else:
+                    _logger.info(f"Content signage failed")
```

Comment:
I think we should change `info` to `warning` or `error`. This will help log aggregation to flag things that we need to wory about
```suggestion
                    _logger.warning(f"Content signage failed")
```

Reply:
Doesn't have to happen this PR just an observation

Reply:
Good catch. Done. Error.